invenio_rdm_records/fixtures/__init__.py

```diff
@@ -41,5 +42,11 @@ def run(self):
             "users.yaml",
         ).load()
 
+        CommunitiesFixture(
+            Path("./app_data/logos"),
+            [Path("./app_data"), dir_ / "data"],
```

Comment:
minor/shelve: something I noticed in other fixtures as well, the `Path("./app_data")` path assumes that the user is running the command from the root cookiecutted directory (or any directory with an `app_data` folder...

The correct approach would be to use `Path(current_app.instance_path) / "app_data"`. This can be addressed separately though, so feel free to shelve.

Reply:
@anikachurilova could you open an issue about this in invenio-rdm-records?